docs/getting-started/navigating-the-toolkit.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Navigating the Toolkit
```

Comment:
I like this section to guide people and explain that there are different ways to use. But I am not happy with the title, "Navigating" does not fit here and it's too long. If we move the installation section as the first entry in "Getting Started", this section can be renamed to "Using the Toolkit" or "Usage Patterns", "Implementation Patterns", something in this direction

Reply:
I agree, I'll name it "Usage Patterns" for now, since this is all there is.

If we add more content in the future we can rename.